targets/TARGET_STM/TARGET_STM32F7/flash_api.c

```diff
@@ -130,6 +132,11 @@ int32_t flash_erase_sector(flash_t *obj, uint32_t address)
         status = -1;
     }
 
+    address_sector = GetSectorBase(SectorId);
+
+    SCB_CleanInvalidateDCache_by_Addr(&address_sector, GetSectorSize(SectorId));
```

Comment:
That still wants to be `address_sector`, not `&address_sector`. Not sure how that's working. Or even just

    SCB_InvalidateDCache_by_Addr((uint32_t *) GetSectorBase(SectorId), GetSectorSize(SectorId));

(Invalidate vs CleanInvalidate doesn't matter - it's a write-through region that we're not writing to anyway)

Bit wonky that the function takes `uint32_t *`, it doesn't actually need a word-aligned address.